flyway/sql/V1.0.0__init.sql

```diff
@@ -1,13 +1,45 @@
 -- テーブル作成
--- ユーザマスタ
-CREATE TABLE m_user
+-- ユーザトランザクション
+CREATE TABLE users
 (
-    id                        int unsigned NOT NULL AUTO_INCREMENT COMMENT 'ID',
-    created_at                datetime            DEFAULT CURRENT_TIMESTAMP COMMENT '登録日時',
-    updated_at                datetime            DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP COMMENT '更新日時',
-    deleted_at                datetime            DEFAULT NULL COMMENT '削除日時',
-    PRIMARY KEY (id)
+    user_id    int unsigned NOT NULL AUTO_INCREMENT COMMENT 'user_id',
+    email      varchar(254) NOT NULL COMMENT 'ログイン用email',
+    password   text         NOT NULL COMMENT 'ログイン用password',
+    created_at datetime DEFAULT CURRENT_TIMESTAMP COMMENT '登録日時',
+    PRIMARY KEY (user_id)
 ) ENGINE = InnoDB
   DEFAULT CHARSET = utf8mb4
-  COLLATE = utf8mb4_unicode_ci comment ='ユーザマスタ'
-;
\ No newline at end of file
+  COLLATE = utf8mb4_unicode_ci
+    COMMENT = 'ユーザトランザクション'
+;
+
+-- 投稿トランザクション
+CREATE TABLE posts
+(
+    post_id int unsigned NOT NULL AUTO_INCREMENT COMMENT 'post_id',
+    user_id int unsigned NOT NULL COMMENT '投稿者user_id',
+    body    text         NOT NULL COMMENT '本文',
+    date    datetime DEFAULT CURRENT_TIMESTAMP COMMENT '投稿日付',
```

Comment:
[nits]
超細かいですが、datetimeなので日付じゃなくて日時ですね

```suggestion
    date    datetime DEFAULT CURRENT_TIMESTAMP COMMENT '投稿日時',
```